.github/workflows/test-pr.yml

```
@@ -61,6 +61,7 @@ jobs:
             generate_tests_kubernetes_apps_default_test_case_enabled: false
             generate_tests_kubernetes_apps_test_case_jupyterhub_enabled: true
             generate_tests_kubernetes_apps_test_case_daskhub_enabled: true
+            flux_enabled: true
```

Let's add a comment here to clarify why this is here
```suggestion
            # Test Flux install in CI even when disabled in default config
            flux_enabled: true
```